docs/content/en/docs/reference/baremetal/bare-custom-hookos.md

```diff
@@ -3,6 +3,168 @@ title: "Customize HookOS for EKS Anywhere on Bare Metal"
 linkTitle: "Customize HookOS"
 weight: 30
 description: >
-  Customizing HookOS to use with EKS Anywhere on Bare Metal
+  Customizing HookOS for EKS Anywhere on Bare Metal
 ---
 
+To initally PXE boot bare metal machines used in EKS Anywhere clusters, Tinkerbell acquires a kernel and initial ramdisk that is referred to as the HookOS.
+A default HookOS is provided when you create an EKS Anywhere cluster.
+However, there may be cases where you want to override the default HookOS, such as to add drivers required to boot your particular type of hardware.
+
+The following procedure describes how to get the Tinkerbell stack’s Hook/Linuxkit OS built locally.
+For more information on Tinkerbell’s Hook Installation Environment, see the [Tinkerbell Hook repo](https://github.com/tinkerbell/hook).
+
+1. Clone the hook repo or your fork of that repo:
+
+    ```bash
+    git clone https://github.com/tinkerbell/hook.git
+    cd hook/
+    ```
+
+1. Pull down the commit that EKS Anywhere is tracking for Hook:
+
+    ```bash
+    git checkout -b <new-branch> 029ef8f0711579717bfd14ac5eb63cdc3e658b1d
+    ```
+
+    >**_NOTE_**: This commit number can be obtained from the [EKS-A build tooling repo](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/tinkerbell/hook/GIT_TAG).
+    >
+
+1. Make changes shown in the following `diff` in the `Makefile` located in the root of the repo using your favorite editor. 
+
+    ```bash
+    diff —git a/Makefile b/Makefile
+    index 66b7f48..f9fc283 100644
+    --- a/Makefile
+    +++ b/Makefile
+    @@ -1,4 +1,4 @@
+    -ORG ?= quay.io/tinkerbell (http://quay.io/tinkerbell)
+    +ORG ?= localhost:5000/tinkerbell
+     ARCH := $(shell uname -m)
+    
+     GIT_VERSION ?= $(shell git log -1 —format="%h")
+    @@ -53,13 +53,13 @@ dev-bootkitBuild:
+     cd bootkit; docker buildx build —load -t $(ORG)/hook-bootkit:0.0 .
+    
+     bootkitBuild:
+    -  cd bootkit; docker buildx build —platform linux/amd64,linux/arm64 —push -t $(ORG)/hook-bootkit:0.0 .
+    +  cd bootkit; docker buildx build —platform linux/amd64 —push -t $(ORG)/hook-bootkit:0.0 .
+    
+     dev-tink-dockerBuild:
+     cd tink-docker; docker buildx build —load -t $(ORG)/hook-docker:0.0 .
+    
+     tink-dockerBuild:
+    -  cd tink-docker; docker buildx build —platform linux/amd64,linux/arm64 —push -t $(ORG)/hook-docker:0.0 .
+    +  cd tink-docker; docker buildx build —platform linux/amd64 —push -t $(ORG)/hook-docker:0.0 .
+    ```
+
+    Changes above change the ORG variable to use a local registry (`localhost:5000`) and change the `docker build` command to only build for the immediately required platform to save time.
+
+
+1. Modify the `hook.yaml` file located in the root of the repo with the following changes:
+
+    ```bash
+    *diff --git a/hook.yaml b/hook.yaml*
```

Comment:
Hmm, what are these asterisks here and down below. They are showing up like that on the rendered page as well, might be a copy/paste thing from quip

Reply:
Yes. I think they are bold markings. I missed that. I'll remove them.

Reply:
Done.